lib/osf-components/addon/components/node-card/template.hbs

```diff
@@ -100,32 +100,79 @@
             </div>
             {{#if @node.isRegistration}}
                 {{#if this.registrationTitle}}
-                    <div local-class='NodeCard__registration-title'>
+                    <div
+                        data-test-registration-title 
+                        local-class='NodeCard__registration-title'
+                    >
                         {{this.registrationTitle}}
                     </div>
                 {{/if}}
-                <div local-class='NodeCard__schema-name'>
+                <div
+                    data-test-schema-name
+                    local-class='NodeCard__schema-name'
+                >
+                    <strong
+                        data-test-schema-label
+                        local-class='Labels'
+                    >
+                        {{t 'node_card.schema'}}
+                    </strong>
                     {{@node.registrationSchema.name}}
                 </div>
             {{/if}}
-            <div local-class='NodeCard__authors'>
+            <div data-test-contributors
```

Comment:
[HTML dl](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) is more suitable for `key-value` type of data. Might also be best for accessibility. It indicates that this is a metadata list for the registration.